### DISCUSSION TITLE
formily/vue 使用 json-schema 方式渲染表单，当 schema 节点的 type=void 时，无法用 x-content 设置插槽

### DISCUSSION THREAD

**Discussion by jiezi19971225** [alibaba/formily#2044] [Q&A 2.x] [answered]
https://codesandbox.io/s/strange-knuth-pretw?file=/src/components/HelloWorld.vue
实际使用场景是用 formily 渲染非表单元素组件，type 为 void，需要通过slot设置内容

这里看了源码 RecursionField 组件的源码，type 为 void 时是没有处理 slot 的

![image](https://user-images.githubusercontent.com/24801265/130738683-2079e5bd-1e22-4784-8656-ea809e5a1881.png)
请问这是特性还是bug

**Comment by MisicDemone** [comment, marked as answer]
嗯，当前的实现下ObjectField和VoidField只支持 default slot，属于实现的问题。欢迎PR

**Reply by jiezi19971225** [reply]
好的 已提pr https://github.com/alibaba/formily/pull/2046
